sdk/eventgrid/Azure.Messaging.EventGrid/tests/Infrastructure/EventGridLiveTestBase.cs

```diff
@@ -12,6 +12,7 @@ public EventGridLiveTestBase(bool isAsync) : base(isAsync/*, RecordedTestMode.Re
             Sanitizer = new RecordedTestSanitizer();
             Sanitizer.SanitizedHeaders.Add(Constants.SasKeyName);
             Sanitizer.SanitizedHeaders.Add(Constants.SasTokenName);
+            Sanitizer.JsonPathSanitizers.Add("$..traceparent");
```

Comment:
Are we sanitizing just to ignore it?

Reply:
Yep.. It would be nice if we had something like JsonPathsToIgnore.